diff -dur opengfx-0.5.0-source.orig/Makefile opengfx-0.5.0-source/Makefile
--- opengfx-0.5.0-source.orig/Makefile	2014-03-31 21:20:26.000000000 +0200
+++ opengfx-0.5.0-source/Makefile	2014-04-08 20:25:25.046918657 +0200
@@ -240,7 +240,7 @@
 #####################################################
 lng: custom_tags.txt
 
-custom_tags.txt: FORCE
+custom_tags.txt:
 	$(_E) "[LNG] $@"
 	$(_V) echo "VERSION        :$(REPO_VERSION_STRING)" > $@
 	$(_V) echo "VERSION_STRING :$(REPO_VERSION_STRING)" >> $@
@@ -532,9 +532,9 @@
 	$(_E) "Try calling 'make install -D INSTALL_DIR=path/to/install_dir'"
 	$(_V) false
 endif
-	$(_E) "[INSTALL] to $(INSTALL_DIR)"
-	$(_V) install -d $(INSTALL_DIR)
-	$(_V) install -m644 $< $(INSTALL_DIR)
+	$(_E) "[INSTALL] to $(DESTDIR)$(INSTALL_DIR)"
+	$(_V) install -d $(DESTDIR)$(INSTALL_DIR)
+	$(_V) install -m644 $< $(DESTDIR)$(INSTALL_DIR)
 
 # misc. convenience targets like 'langcheck'
 -include $(SCRIPT_DIR)/Makefile_misc
